third_party/packages/ansi_up/lib/ansi_up.dart

```diff
@@ -297,8 +299,8 @@ enum PacketKind {
   OSCURL,
 }
 
-class TextPacket {
-  TextPacket({this.kind, this.text, this.url});
+class _TextPacket {
```

Comment:
why was this made private?

Reply:
From what I could tell, it looked like a class internal to this package. I can change back to public if that is incorrect

Reply:
leave it as private. I think you are right.